app/src/main.js

```diff
@@ -0,0 +1,75 @@
+import Vue from 'vue';
+import VueApexCharts from 'vue-apexcharts';
+
+import App from './App.vue';
+import './plugins/vuetify';
+import router from './router';
+
+import oasis from '@oasislabs/client';
+
+let SecretBallot;
+
+// TODO: Eliminate wallet and transition to Developer Gateway
+// Wallet private key.
+const privateKey = '<private key>';
+
+// Wallet for signing and paying for transactions.
+const wallet = new oasis.Wallet(privateKey);
+
+// Ethereum gateway responsible for signing transactions.
+const gateway = new oasis.gateways.Web3Gateway('wss://web3.devnet.oasiscloud-staging.net/ws', wallet);
+
+oasis.setGateway(gateway);
+
+// Ballot Instantiation
+window.deployService = async (constructorArgs) => {
+  const bytecode = await fetch('/assets/bytecode/ballot.wasm')
```

Comment:
ideally, we wouldn't include built files in the repo source, we can include it when we publish as a built task, but otherwise this binary will change every time there's changes to the source code, making and diffs of that hard to review